### ISSUE TITLE
[BUG] Warp Rifle description error

### ISSUE THREAD

**Issue by compcon** [massif-press/compcon#2022]
<strong>Submitted At: </strong> 5/28/2022 13:47:46<br><br><strong>Source: </strong> The Long Rim<br><br><strong>Incorrect Text/Data: </strong><br> EQUIPMENT EFFECT

The target must pass an ENGINEERING save or be teleported a number of spaces equal to the damage you dealt with this weapon (including bonus damage, etc.; no more than 10 spaces), in a direction of your choice. They must end in a free, valid space.<br><br><strong>Correct Text/Data: </strong><br> On Hit: The target must pass an ENGINEERING save or
be teleported a number of spaces equal to the
damage you dealt with this weapon (including
bonus damage, etc.; no more than 10 spaces), in a
direction of your choice. They must end in a free,
valid space.
<br>

**Comment by msprijatelj** [comment]
Makes sense; this should be in an `on_hit` field and not a general `effect` field.  Thankfully, in most cases, a PC would need to hit to deal damage at all, but this could be relevant for cases where "miss" damage occurs (or if Reliable somehow finds its way to this weapon).  I'll move this to The Long Rim repo.
